<ui version="4.0" >
 <class>XMLDB_MakeDatClass</class>
 <widget class="QWidget" name="XMLDB_MakeDatClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>XMLDB_MakeDat</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>IST TAG</string>
   </property>
   <widget class="QPushButton" name="btnMkIst" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;IST ADD</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>360</x>
      <y>50</y>
      <width>95</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>NGICnt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnNgCnt_2" >
       <property name="maximum" >
        <number>9999</number>
       </property>
       <property name="singleStep" >
        <number>10</number>
       </property>
       <property name="value" >
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>242</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>LotID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnLotIt" >
       <property name="maximum" >
        <number>9999999</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>INSPECTID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnEID" >
       <property name="maximum" >
        <number>99999999</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Ver</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtVer" >
       <property name="text" >
        <string>P-111</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnCnt" >
       <property name="minimum" >
        <number>10</number>
       </property>
       <property name="maximum" >
        <number>9999999</number>
       </property>
       <property name="singleStep" >
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>50</y>
      <width>95</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>NGPCnt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnNgCnt" >
       <property name="maximum" >
        <number>9999</number>
       </property>
       <property name="singleStep" >
        <number>10</number>
       </property>
       <property name="value" >
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>80</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>DbFile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDbFile_2" >
       <property name="text" >
        <string>D:\Data\xmldb.fdb</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExit" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>230</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMess" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string>CreateDatabase</string>
   </property>
   <widget class="QPushButton" name="btnCreate" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Crerate</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>DataBaseFileName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDbFile" >
       <property name="text" >
        <string>D:\Data\xmldb.fdb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDbFile" >
       <property name="text" >
        <string>&amp;SelFile</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
